nncf/common/graph/layer_attributes.py

```diff
@@ -19,6 +19,7 @@
 class Dtype(Enum):
     FLOAT = "float"
     INTEGER = "int"
+    STRING = "string"
```

Comment:
may you have test for it?

Reply:
I have added tests to be sure that we cover all available OpenVINO data types.